pkg/pod_lister/pod_lister_test.go

```diff
@@ -0,0 +1,78 @@
+package pod_lister
+
+import (
+	"os"
+	"testing"
+
+	. "github.com/onsi/gomega"
+)
+
+func createTempFile(contents string) (filename string, reterr error) {
+	f, err := os.Create("/tmp/987654321")
+	if err != nil {
+		return "", err
+	}
+	defer func() {
+		if err := f.Close(); err != nil {
+			return
+		}
+	}()
+	_, _ = f.WriteString(contents)
+	return f.Name(), nil
+}
+
+const rhel_containerd = `
```

Comment:
can you add the other two cases [here](https://github.com/sustainable-computing-io/kepler/pull/152#issue-1367220950)